src/csync/CMakeLists.txt

```diff
@@ -46,8 +46,8 @@ if(WIN32)
         PUBLIC __USE_MINGW_ANSI_STDIO=1
                NOMINMAX
                # Get APIs from from Vista onwards.
-               _WIN32_WINNT=0x0600
-               WINVER=0x0600
+               _WIN32_WINNT=_WIN32_WINNT_WIN8
+               WINVER=_WIN32_WINNT_WIN8
                NTDDI_VERSION=0x0A000003
```

Comment:
I guess we should set those in https://github.com/owncloud/client/blob/master/cmake/modules/OCApplyCommonSettings.cmake#L6 

Also the comment should change I guess :D

NTDDI_VERSION=NTDDI_WIN10_RS2 see https://learn.microsoft.com/en-us/windows/win32/winprog/using-the-windows-headers?redirectedfrom=MSDN

Reply:
done